concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -1,197 +1,362 @@
 #!/usr/bin/env python3
-
+import argparse
 import os
 import sys
 import datetime
 import hashlib
-import time
 import random
-from base64                 import b64encode
+import requests
+import time
 
-from golem_messages         import message
-from golem_messages         import shortcuts
+from base64                          import b64encode
+from golem_messages                  import message
+from golem_messages                  import shortcuts
+from golem_messages.message.concents import AckForceGetTaskResult
+from golem_messages.message.concents import ForceGetTaskResultFailed
+from golem_messages.message.concents import ForceGetTaskResultUpload
 
-from utils.helpers          import get_current_utc_timestamp
-from utils.testing_helpers  import generate_ecc_key_pair
+from utils.helpers                   import get_current_utc_timestamp
+from utils.testing_helpers           import generate_ecc_key_pair
 
-from api_testing_helpers    import api_request
-from api_testing_helpers    import timestamp_to_isoformat
-
-from freezegun              import freeze_time
-import requests
+from api_testing_common              import api_request
+from api_testing_common              import create_task_to_compute
+from api_testing_common              import get_protocol_constants
+from api_testing_common              import print_protocol_constants
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
 
-(PROVIDER_PRIVATE_KEY,  PROVIDER_PUBLIC_KEY)  = generate_ecc_key_pair()
-(REQUESTOR_PRIVATE_KEY, REQUESTOR_PUBLIC_KEY) = generate_ecc_key_pair()
-
+DEADLINE_OFFSET        = 5
+CONCENT_MESSAGING_TIME = 30
+FORCE_ACCEPTANCE_TIME  = 30
+TOKEN_EXPIRATION_TIME  = 3600
 
-def parse_command_line(command_line):
-    if len(command_line) <= 1:
-        sys.exit('Not enough arguments')
+WAIT_TIME_1            = DEADLINE_OFFSET + FORCE_ACCEPTANCE_TIME
```

Comment:
Please give it a more meaningful name. When I was reading the test code I had to go up here to check what this is. If it had a self-explanatory name I would not have to do so.

Actually, any good name I can come up with is the combination of those two so I think that we should remove this constant. It would be more self-explanatory to have just `DEADLINE_OFFSET + FORCE_ACCEPTANCE_TIME` in the code.

Reply:
a more meaningful name has been given